news/8.bugfix

```diff
@@ -0,0 +1 @@
+The http status of the response is changed from 301 (Moved Permanently) to 302 (Found) for GET requests and to 307 (Temporary Redirect) for other request methods because nothing prevents the URL to be reused in the future. [ale-rt]
```

Comment:
I'm not sure whether this shouldn't be considered a breaking change (instead of a bugfix). Sure, one could argue that the existing behavior was incorrect, and now gets fixed. But it also does have some potential of breaking client apps in subtle ways.